NetKVM/Common/ParaNdis-Common.cpp

```diff
@@ -1841,7 +1841,28 @@ void ParaNdis_ReuseRxNBLs(PNET_BUFFER_LIST pNBL)
     }
 }
 
-bool ParaNdis_DPCWorkBody(PARANDIS_ADAPTER *pContext, ULONG ulMaxPacketsToIndicate)
+void ParaNdis_CXDPCWorkBody(PARANDIS_ADAPTER *pContext)
+{
+    InterlockedIncrement(&pContext->counterDPCInside);
+    if (pContext->bEnableInterruptHandlingDPC && pContext->CXPath.WasInterruptReported())
```

Comment:
bEnableInterruptHandlingDPC - the name is confusing...

Reply:
This is an existing flag, I did not add it. Moreover i think it shouldn't be used, it is a leftover.